tensorrt_llm/_torch/auto_deploy/transform/library/sharding.py

```
@@ -3006,21 +3223,50 @@ def _shard_qk_norm(
             ad_logger.debug(
-                f"Added RMSNormShardingInfo for {user_node.name} "
-                f"(will replace with sharded_rmsnorm for global mean)"
+                f"Added {info_cls.__name__} for {user_node.name} (strategy: {strategy_label})"
             )
             added_nodes += 1
 
+    # Register paired Q+K fused transforms when use_fused_qk_norm is active.
+    if config.use_fused_qk_norm and qk_norm_candidates:
+        q_norms = [(n, w) for n, w in qk_norm_candidates if "q_proj" in w]
+        k_norms = [(n, w) for n, w in qk_norm_candidates if "k_proj" in w]
+        for (q_node, _), (k_node, _) in zip(q_norms, k_norms):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)